src/core/utils.ts

```diff
@@ -170,6 +205,65 @@ export function getChartLegendItems(chart: Highcharts.Chart): readonly LegendIte
   return legendItems;
 }
 
+export function getChartLegendItemsFromSeriesOptions(
```

Comment:
This util (and probably other changes in this file) won't be needed if the standalone legend would accept items instead of HC series.